src/test/resources/test_data/subject-access-request-report-id-4.sql

```diff
@@ -21,7 +21,7 @@ VALUES
     'subject-access-request-report',
     'pdf',
     21384,
-    'd58e3582afa99040e27b92b13c8f2280',
+    '',
```

Comment:
Why did this need changing?

Reply:
The TL;DR is laziness with a hint of logic...

Logic: When documents are added the file hash isn't actually calculated or persisted so the tests were asserting it was returned on a GET and then asserting it was not populated on a POST which seemed illogical. 

Laziness: Removing this meant I could use the same assertion for both the GET and POST document requests.

However now I'm writing this out I feel like I should just pass the expected filehash into the assertion...